app/main.py

```
@@ -0,0 +1,41 @@
+    try:
+        Base.metadata.create_all(bind=engine)
+        print("Database tables created successfully")
+    except OperationalError as e:
+        print(f"Database connection failed: {e}")
+
+@app.get("/")
+def read_root():
+    return {"Hello": "World", "message": "Ticket Management API is running"}
+
+@app.post("/tickets", response_model=TicketResponse)
+def evaluate_ticket(ticket: TicketCreate, db: Session = Depends(get_db)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)